### ISSUE TITLE
Re-sort when a comment on a post is made

### ISSUE THREAD

**Issue by galenwp** [urbit/urbit#4189]
**Describe the bug**
When I comment on a post in a notebook, that notebook doesn't re-sort in the sidebar

**To Reproduce**
1. Go to a notebook
2. Make a comment
3. See it remain in the same place on the sidebar

**Expected behavior**
If a comment doesn't cause an unread / re-sort on a notebook, I'll never know if any activity is happening in there.

**Comment by matildepark** [comment]
Are you sure you want to resort the sidebar based on any last updated child nodes of the graph, not just the parent nodes? That could get busy and confusing — you'll often see a channel jump to the top and have to go looking for the new comments down in the list indefinitely. 

Comments really only matter when you're subscribed to the thread for notifications, IMO.

Tagging @urcades — a UX question, I suppose.

**Comment by urcades** [comment]
> That could get busy and confusing — you'll often see a channel jump to the top and have to go looking for the new comments down in the list indefinitely.

Yeah, I was evaluating this when it was first proposed and came to a similar conclusion — while the issue might not be "too much of an issue" these days, I optimistically imagine a future where comments on child elements in a channel are frequent and heavy, and the navigational leap from 'sidebar reorder' to 'some child element' can become quickly confusing as the interfaces currently stand.

I think this could be mitigated via more granular/specific filtering of inboxed updates, but I still need to think about this.

tl;dr: The **current** interface doesn't really support a legible discovery mechanism of updated posts to a channel, so this issue **as-is** is something we probably don't want to do currently.

**Comment by matildepark** [comment]
So, not an issue — but a design consideration in a broader sense for later. Closing.
